design-documents/graph-ql/coverage/product-reviews.graphqls

```diff
@@ -65,12 +68,17 @@ type CreateProductReviewOutput {
 input CreateProductReviewInput {
     sku: String! @doc(description: "The SKU of the product that the review is assigned")
     nickname: String! @doc(description: "The customer's nickname. Defaults to customer name if logged in.")
-    summary: String! @doc(description: "The review summary (a.k.a title")
-    text: String! @doc(description: "The review text.")
-    ratings: [ProductReviewRatingInput!]! @doc(description: "Ratings details by category. e.g price: 5, quality: 4 etc")
+    title: String! @doc(description: "The review title")
```

Comment:
Based on https://github.com/magento/architecture/pull/343#discussion_r461735711 changes in these two lines should be reverted.